### PR TITLE
Update convert_prompt.py

### PR DESCRIPTION
**Pull request by yeganehkordi**
I've updated the code. Please let me know if I missed sth.
changes:
- Adding subset selection for each dataset
- Adding prompt name and id to the json files

### CODE REVIEW

File: src/convert_prompt.py [hunk 1]
```diff
@@ -39,15 +39,19 @@
 
 def get_dataset(dataset_name):
     dataset = load_dataset(dataset_name, split="train")
+    cap = min(6500, len(dataset))
```
Comment (danyaljj) [inline src/convert_prompt.py]: is it possible to make this an optional parameter? 
Basically, we don't want to sample by default, unless `--sample 6500` is given in the input.

Comment (yeganehkordi) [reply src/convert_prompt.py]: Ya, I'll fix it now.

**Comment by yizhongw** [comment]
This conversion script looks good to me!

**Comment by yizhongw** [comment]
Question: @yeganehkordi where are you going to put the shared task lists between NI and PromptSource?

**Comment by yeganehkordi** [comment]
> Question: @yeganehkordi where are you going to put the shared task lists between NI and PromptSource?

I'll update the spreadsheet now and list the shared tasks there.

**Comment by danyaljj** [comment]
> I'll update the spreadsheet now and list the shared tasks there. 

It might be easier if you share the list as a json file in a different PR. This way, any followup work can also use this mapping for conducting further research.

**Comment by yeganehkordi** [comment]
> > I'll update the spreadsheet now and list the shared tasks there.
> 
> It might be easier if you share the list as a json file in a different PR. This way, any followup work can also use this mapping for conducting further research.

Sounds good! Will do.

**Pull request merged**